interpreter/exec/v128.ml

```diff
@@ -1,9 +1,8 @@
 include Simd.Make
   (struct
-    include Bytes
+    include String
     let bytewidth = 16
-    let of_bits = Bytes.of_string
-    let to_bits = Bytes.to_string
+    let to_string s = s
```

Comment:
This def seems unnecessary?

Reply:
String has no `to_string` function, so this struct doesn't match RepType if we remove this.

Reply:
Oh, I see. But I would suggest to keep the name `to_bits` for this, to avoid confusion with pretty printing.

Reply:
to_bits is still around, it is defined inside Simd.Make.
This to_string is for pretty print, and if you look at Simd.Make's `to_string`, you can see a FIXME comment, saying that this implementation is very wrong :)
Perhaps I should copy the FIXME here to make it clear that this should be fixed?

Reply:
Yes, I was wondering about the FIXME. I suppose I don't quite see how a fixed implementation would use Rep.to_string instead of doing its own thing on the "bits". So I would remove it until it's actually clear it's needed. But it's all a minor point, so feel free to leave it alone.

Reply:
Sounds good, I'll leave it alone for now. We can probably change this to print hex values instead. Thanks!